src/v1/constants/index.js

```diff
@@ -0,0 +1,4 @@
+export * as collectionsEnum from "./collectionsEnum.js";
+//export * from './errors.js';
```

Comment:
This will need to be uncommented if custom errors are ever desired in an implementation of this template